debt_fraction_calculator/debt_fraction_calc.py

```diff
@@ -230,7 +231,7 @@ def calculate_all_debt_fractions(data_workbook_filename: str, output_filename: s
     OUTPUT_FILENAME - File to save calculated debt fractions to. Should end with .csv
     """
     tech_map: Dict[str, Type[TechProcessor]] = {tech.__name__: tech for tech in LCOE_TECHS}
-    techs = LCOE_TECHS if tech is None else [tech_map[tech]]
+    techs = [lcoe_calculator.tech_processors.UtilityPvPlusBatteryProc]
```

Comment:
Is this correct?

Reply:
It is not. Looks like I forgot to update this file after getting things working in lcoe calculator. I'll convert the PR to draft.